.github/workflows/test-migrations-two-step.yml

```
@@ -77,6 +77,13 @@ jobs:
       - name: Create database
         run: './bin/rails db:create'
 
+      - name: Restore DB as of `20160220174730`
+        run: |
+          mv db/schema.rb db/schema-latest.rb
+          cp db/schema/snapshots/20160220174730.rb db/schema.rb
+          ./bin/rails db:schema:load
+          mv -f db/schema-latest.rb db/schema.rb
```

The following should work:
```suggestion
      - name: Restore DB as of `20160220174730`
        run: |
          SCHEMA=db/schema/snapshots/20160220174730.rb ./bin/rails db:schema:load
```

---

Ah! Thank you -- I swore this feature existed but could not find it documented, but it's definitely there -- https://github.com/rails/rails/blob/7-1-stable/activerecord/lib/active_record/tasks/database_tasks.rb#L428

That approach is much cleaner, will update PR.

---

Updated PR to use `SCHEMA` env var to load. Much cleaner.